package/yast2-firewall.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Fri Mar 23 11:38:40 UTC 2018 - knut.anderssen@suse.com
+
+- Do not support FW_LOG_ACCEP_CRIT tag as firewalld only log
```

Comment:
typo + I don't understand the message anyway -> going to read it one more time ;-)